e2e/.#rdctl.e2e.spec.ts

```diff
@@ -0,0 +1 @@
+ericp@chirico.local.14112
```

Comment:
This probably shouldn't be checked in. (You'll probably want to modify git history to get rid of it.)

Reply:
Thanks. Think it's gone. Not sure how that file got created.